spec/system/admin/all_transactions_search_spec.rb

```diff
@@ -27,14 +27,15 @@
 
     select_from_chosen accounts.second.account_list_item, from: "Payment Sources"
     click_button "Filter"
-    
+
     expect(page).to have_content("Total")
-    expect(page).not_to have_link(order_detail.id.to_s)
-    expect(page).to have_link(orders.second.order_details.first.id.to_s)
+    expect(page).not_to have_link(order_detail.id.to_s, href: manage_facility_order_order_detail_path(order_detail.facility, order_detail.order, order_detail))
+    expect(page).to have_link(orders.second.order_details.first.id.to_s, href: manage_facility_order_order_detail_path(orders.second.facility, orders.second, orders.second.order_details.first))
+    # manage-order-detail
```

Comment:
Can you add some more detail to this comment?

Reply:
Oh, this shouldn't be here 😅